app/models/policy.rb

```diff
@@ -20,4 +31,8 @@ def validate_schema_specification
     return if specification.valid?
     specification.errors[:base].each { |error| errors.add(:schema, error) }
   end
+
+  def set_identifier
+    self.identifier = "#{name}-#{version}".gsub(/\s/, '-')
```

Comment:
This needs to be documented. The identifier is actually `name-version` with spaces replaced with dashes.

Reply:
Why replacing space with dash? I do not see any reason for that

Reply:
@hallelujah I think that it is this way for the search later by path in the url for the show request containing this identifier 😄

Reply:
Well URl can have spaces with `%20`

Reply:
Removed.